test/unit/resolveurl.html

```diff
@@ -138,6 +147,13 @@
         assert.equal(el.$.hash.getAttribute('href'), '#foo.z', 'hash-only url should not be resolved');
         assert.equal(el.$.absolute.getAttribute('href'), '/foo.z', 'absolute urls should not be resolved');
         assert.equal(el.$.protocol.getAttribute('href'), 'data:foo.z', 'urls with other protocols should not be resolved');
+        el.$.if.render();
```

Comment:
Would be an improvement to fix the `rx` to help ensure the url is fully resolved correctly and not just that it contains the `sub` folder.